example/text_classification/README.md

```diff
@@ -1,14 +1,22 @@
-# Text Classification Example 
+# Text Classification Example
 
 |      | Android | iOS | Linux | Mac | Windows | Web |
 |------|---------|-----|-------|-----|---------|-----|
-| file | ✅       | ✅   |       | ✅   |    ✅    |     |
-
-Example for using esrgan in flutter with TF Lite
-Refer to [Medium Blog](https://medium.com/@am15hg/text-classification-using-tensorflow-lite-plugin-for-flutter-3b92f6655982) for a detailed tutorial.
+| file | ✅       | ✅   | ✅   | ✅   |    ✅    |     |
 
 ## Overview
 
 This basic Text Classification app demonstrates the usage of tflite_flutter_plugin.
 
 ![DEMO GIF](demo.gif)
+
+## Setup
+
+Run `sh scripts/download_model.sh` to download models.
+
+## Supported platforms
```

Comment:
"Supported platforms" overlap. I think we should remove this.

Reply:
I have resolved this issue. Thanks.

Reply:
Yes you are right, thank you for resolving this!